play_tester.py

```diff
@@ -10,9 +10,10 @@
 
 # play testing section
 print("\n-----Wave 01 test data-----")
-pp.pprint(HORROR_1)
-pp.pprint(FANTASY_1)
-pp.pprint(FANTASY_2)
+print(create_movie(MOVIE_TITLE_1,GENRE_1,RATING_1))
```

Comment:
Just a little FYI, in the future, if you make changes to a sandbox file like this, you don't need to add it to the staging area/commit it and push up the changes.

It's no problem that you did so here since this isn't a PR that will get merged into the project, but if your changes were to be merged into a project then someone reviewing your code would probably ask you to remove these changes from the PR so they don't get merged into the main code base.